src/auth/screens/events.screen.js

```diff
@@ -155,7 +155,7 @@ class Events extends Component {
             'auth.events.pullRequestReviewCommentEvent',
             language,
             {
-              action: translate(`auth.events.actions.${action}`, language),
+              action: translate('auth.events.actions.commented', language),
```

Comment:
I'm assuming it's incorrect because passing in `created` is why the wrong message shows right? Perfectly okay with having `.commented` specifically here in that case